src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -366,6 +366,13 @@ std::string ContextAttribute::renderAsJsonObject
         valueIsNumberOrBool = true;
         break;
 
+#if 0
```

Comment:
@kzangeli , this is something that I feel you could need in you next PR. In negative case, remove this block upon next PR completion.

NTC